Lib/ast.py

```diff
@@ -148,6 +154,13 @@ def _format(node, level=0):
                 if value is None and getattr(cls, name, ...) is None:
                     keywords = True
                     continue
+                if (
+                    not show_empty
+                    and value in empty_values
```

Comment:
```suggestion
                    and (value is None or value == [])
```
I don't think the `empty_values` variable helps readability here